feedthefox/users/urls.py

```diff
@@ -7,12 +7,15 @@
 
 
 account_patterns = [
+    # TODO: remove signup
     url(r'^signup/?$', socialaccount_views.signup, name='socialaccount_signup'),
+    url(r'^inactive/?$', account_views.account_inactive, name='account_inactive'),
+    # TODO: fix this
+    url(r'^connections/?$', socialaccount_views.connections, name='socialaccount_connections')
```

Comment:
To open the flow to users to connect other accounts when they are logged in. I am going to remove this from here and open an issue.